include/rmm/mr/device/logging_resource_adaptor.hpp

```diff
@@ -19,9 +19,16 @@
 
 #include <rmm/detail/error.hpp>
 
+// If using GCC, temporary workaround for older libcudacxx defining _LIBCPP_VERSION
+// undefine it before including spdlog, due to fmtlib checking if it is defined
+// TODO: remove once libcudacxx is on Github and RAPIDS depends on it
+#ifdef __GNUG__
+#undef _LIBCPP_VERSION
```

Comment:
No. This is defined by clangs libc++, which we don't use (GCC's is libstdc++). It's an accident that it gets defined by the version of libcudacxx that we use, because it is based on libc++. THe latest libcudacxx fixes this, but it's only available in the CUDA 11 TK and later, so we can't use it. We are waiting for libcudacxx to go to Github so we can use it on earlier CUDA versions.

Reply:
Shouldn't we change this macro name in our freestanding fork?